sqldb/sqlc/migrations/000006_invoice_migration.up.sql

```diff
@@ -0,0 +1,14 @@
+-- invoice_payment_hashes table contains the hash of the invoices. This table
+-- is used during KV to SQL invoice migration as in our KV representation we
+-- don't have a mapping from hash to add index.
+CREATE TABLE IF NOT EXISTS invoice_payment_hashes (
```

Comment:
It is deleted now, but I wonder why you decided to keep the table and just delete the rows instead of dropping the table ?

Reply:
if we go with duplicating DB state (sql files) and other codecs etc per migration (like we do for our channeldb migrations today) then we would be able to do this no? it would just mean having some duplication. but it might be worth it so that we dont have to have migration methods on the interface and so that we actually can drop these DBs and keep this "live" version clean.

Reply:
right now, the underlying structure of the database does not change with this migration. But in future we may have migration that changes the underlying structure and so at that point, the queries anyways would change and not line up and so I think at that point we any ways would need to do this code duplication

Reply:
Agreed, I think it'd be nice to split this to a follow up PR so we keep scope focused.